services/catalog/rs_server_catalog/user_catalog.py

```diff
@@ -694,7 +707,10 @@ async def manage_put_post_request(  # pylint: disable=too-many-branches
             elif "items" in request.scope["path"]:
                 # try to get the item if it is already part from the collection
                 item = await self.get_item_from_collection(request)
+                # !!!
+                # Removed since RSPY 326, no need for bucket movement
                 content = self.update_stac_item_publication(content, request, item)
```

Comment:
Same, doesn't seem to be removed

Reply:
See this : https://github.com/RS-PYTHON/rs-server/pull/589#discussion_r1812622750